__tests__/comments.test.js

```diff
@@ -61,4 +53,78 @@ describe('GET /api/articles/:article_id/comments', () => {
             expect(response.body.msg).toBe('Bad request');
         });
     });
+})
+
+describe('POST /articles/:article_id/comments', () => {
+
+    test('201: responds with the posted comment', () => {
+        const newComment = {
+            username: 'butter_bridge',
+            body: 'This is a new comment'
+        };
+
+        return request(app)
+        .post('/api/articles/1/comments')
+        .send(newComment)
+        .expect(201)
+        .then((response) => {
+            const comment = response.body.comment;
+            expect(comment).toEqual(
+                expect.objectContaining({
+                    comment_id: expect.any(Number),
+                    article_id: expect.any(Number),
+                    author: 'butter_bridge',
+                    body: 'This is a new comment',
+                    votes: 0,
+                    created_at: expect.any(String)
+                })
+            )
+        })
+
+    })
+
+    test('404: responds with an error message when article is not found', () => {
+        const newComment = {
+            username: 'butter_bridge',
+            body: 'This is a new comment'
+        };
+
+        return request(app)
+        .post('/api/articles/99999/comments')
+        .send(newComment)
+        .expect(404)
+        .then((response) => {
+            expect(response.body.msg).toBe('Article not found');
+        });
+    })
+
+
+    test('400: responds with an error message when article_id is invalid', () => {
+        const newComment = {
+            username: 'butter_bridge',
+            body: 'This is a new comment'
+        }
+
+        return request(app)
+        .post('/api/articles/not-a-number/comments')
+        .send(newComment)
+        .expect(400)
+        .then((response) => {
+            expect(response.body.msg).toBe('Bad request');
+        });
+    })
+
+    test('400: responds with an error message when username or body is missing', () => {
```

Comment:
Your test says it should be a 400 when username or body is missing but then you have only tested one of these scenarios. Please test for when the body is missing too.